source/_integrations/plugwise.markdown

```diff
@@ -89,151 +89,144 @@ This integration follows standard integration removal. No extra steps are requir
+Available actions to all climate gateways: `climate.set_preset_mode`, `climate.set_temperature`, `climate.set_hvac_mode`,
 
-```yaml
-# Example script change the temperature
-script:
-  force_adam_update:
-    sequence:
-      - action: homeassistant.update_entity
-        target:
-          entity_id: climate.living_room
-```
+Additional actions available to Adam: `climate.turn_on`, `climate.turn_off`, `climate.toggle`
 
-#### Reboot the Plugwise gateway
+{% note %}
+The additional actions will change Adam's **regulation mode** (i.e., the HVAC system mode) to either on or off, affecting the operation of **all** connected thermostats. Turning on activates the previously selected heating or cooling mode.
+{% endnote %}
 
-action: `button.press`
+### Schedule select action
 
-```yaml
-# Example script change the thermostat schedule
-script:
-  reboot_gateway:
-    sequence:
-      - action: button.press
-        target:
-          entity_id: button.adam_reboot
-```
+Available action: `select.select_option`
 
-#### Set HVAC mode
+{% tip %}
+The available schedules depend on the [schedules](#schedule_management) you have configured.
+{% endtip %}
 
-action: `climate.set_hvac_mode`
+### HVAC modes
 
-Available options include `off` (Adam only) `auto`, `cool`, `heat`, and `heat_cool` (Anna with Elga only).
+The following HVAC modes are available:
 
-The meaning of `off` is that the Adam regulation is set to off. This means that the connected HVAC-system does not heat or cool, only the domestic hot water heating function, when available, is active.
+- `auto`: Active schedule – The thermostat changes presets/setpoints according to the schedule.
+- `cool or heat`: No active schedule – The system is manually set to cooling or heating mode, activating based on room temperature relative to the thermostat setpoint.
 
-The meaning of `cool` or `heat` is that there is no schedule active. For example, if the system is manually set to cooling- or heating-mode, the system will be active if the room temperature is above/below the thermostat setpoint.
+For Adam:
 
-The meaning of `heat/cool` is that there is no schedule active. For example, if the system is in automatic cooling- or heating-mode, the active preset or manually set temperature is used to control the HVAC system.
+- `off`: Regulation is set to off – The HVAC system does not heat or cool, but the domestic hot water heating function (if available) remains active.
 
-The meaning of `auto` is that a schedule is active and the thermostat will change presets/setpoints accordingly.
+If you have an Anna with Elga:
 
-The last schedule that was active is determined the same way long-tapping the top of Anna works.
+- `heat_cool`: No active schedule – The system is in automatic cooling or heating mode, using the active preset or manually set temperature to control the HVAC system.
```

Comment:
Only Anna(+Elga) supports the `heat_cool` hvac_mode (automatic switching between heating and cooling).
The Adam only supports the separate modes because switching to and from cooling needs to be done manually by the User.

Reply:
That what it says:

```
If you have an Anna with Elga:

- `heat_cool`: No active schedule – The system is in automatic cooling or heating mode, using the active preset or manually set temperature to control the HVAC system.
```

Reply:
Assuming your approval indicates we can resolve this (please reopen/remove approval)